### PR TITLE
Add support for 169.254.(1-254).x (DHCP IPs) + fd00::/8 + fe80::/10

### PR DESCRIPTION
**Pull request by Naouak**
Fix issue #1

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,14 +8,25 @@ module.exports = isLocalIp
 // 172.16.0.0 - 172.31.255.255
 // 192.168.0.0 - 192.168.255.255
 var local = /^(::f{4}:)?(10\.\d{1,3}|127\.\d{1,3}|172\.1[6-9]|172\.2\d|172\.3[0-1]|192\.168)\.\d{1,3}\.\d{1,3}/
+// Matches
```
Comment (DylanPiercey) [inline index.js]: Would it be possible to merge these three new regexes into the existing 'local' regex for consistency?

File: README.md [hunk 2]
```diff
@@ -23,11 +23,13 @@ isLocal("184.55.123.2"); // false
...
 * `::`
 * `::1`
-* `fe80::1`
+* `fe80::/10`
+* `fc00::/7`
 * `10.0.0.0` - `10.255.255.255`
 * `127.0.0.0` - `127.255.255.255`
 * `172.16.0.0` - `172.31.255.255`
 * `192.168.0.0` - `192.168.255.255`
+* `169.254.1.0` - `169.254.254.255`
```
Comment (DylanPiercey) [inline README.md]: Could you sort this numerically? (It is killing me for some reason :p).

IE: 169 before 172 and fc00 before fe80

**Review by DylanPiercey — COMMENTED** [review]
Thanks for the PR! Theres just a few small changes I'd like to see if you have time. If not I'll probably merge this anyways and fix myself however it could take me a couple weeks.

File: index.js (hunk 1 above)
Comment (Naouak) [reply index.js]: I would prefer the opposite. The local regex is bit too hard to read to be easily maintainable. That's why I put each new ones into a new regex.

File: README.md (hunk 2 above)
Comment (Naouak) [reply README.md]: I will do it tomorrow!

File: index.js (hunk 1 above)
Comment (DylanPiercey) [reply index.js]: It's currently just setup as a bunch of 'or's in a regex, however I'm personally fine either way as long is it's consistent. IE, each portion has its own regex or all in one regex joined by ors.

Comment (Naouak) [reply index.js]: I will then split each portion into a regex each tomorrow.

**Comment by Naouak** [comment]
Fixed all code reviews propositions.

**Comment by DylanPiercey** [comment]
Thanks for your work on this!

**Pull request merged**